quickstart/resources/lang/vi/validation.php

```diff
@@ -0,0 +1,185 @@
+    'unique'      => 'Trường :attribute đã có trong cơ sở dữ liệu.',
+    'uploaded'    => 'Trường :attribute tải lên thất bại.',
+    'url'         => 'Trường :attribute không giống với định dạng một URL.',
+    'uuid'        => 'Trường :attribute phải là một chuỗi UUID hợp lệ.',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify custom validation messages for attributes using the
+    | convention "attribute.rule" to name the lines. This makes it quick to
+    | specify a specific custom language line for a given attribute rule.
+    |
+    */
+
+    'custom' => [
+        'attribute-name' => [
+            'rule-name' => 'custom-message',
+        ],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Custom Validation Attributes
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used to swap attribute place-holders
+    | with something more reader friendly such as E-Mail Address instead
+    | of "email". This simply helps us make messages a little cleaner.
+    |
+    */
+
+    'attributes' => [
+        'name'                  => 'tên',
+        'username'              => 'tên đăng nhập',
+        'email'                 => 'email',
+        'first_name'            => 'tên',
+        'last_name'             => 'họ',
+        'password'              => 'mật khẩu',
+        'password_confirmation' => 'xác nhận mật khẩu',
+        'city'                  => 'thành phố',
+        'country'               => 'quốc gia',
+        'address'               => 'địa chỉ',
+        'phone'                 => 'số điện thoại',
+        'mobile'                => 'di động',
+        'age'                   => 'tuổi',
+        'sex'                   => 'giới tính',
+        'gender'                => 'giới tính',
+        'year'                  => 'năm',
+        'month'                 => 'tháng',
+        'day'                   => 'ngày',
+        'hour'                  => 'giờ',
+        'minute'                => 'phút',
+        'second'                => 'giây',
+        'title'                 => 'tiêu đề',
+        'content'               => 'nội dung',
+        'body'                  => 'nội dung',
+        'description'           => 'mô tả',
+        'excerpt'               => 'trích dẫn',
+        'date'                  => 'ngày',
+        'time'                  => 'thời gian',
+        'subject'               => 'tiêu đề',
+        'message'               => 'lời nhắn',
+        'available'             => 'có sẵn',
+        'size'                  => 'kích thước',
+        'entry'                 => 'mục từ',
+        'comment'               => 'nội dung'
```

Comment:
```suggestion
        'comment' => 'nội dung',
```
sửa tương tự với các dòng khác nhé